.changelog/20261.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+api: Fixed a bug where `AllocDirStats` field was missing from Reads Stats client API
```

Comment:
I suppose "Read" should be singular, and it's capitalized just to match the section heading in [api-docs client page](https://developer.hashicorp.com/nomad/api-docs/client#read-stats)?

```suggestion
api: Fixed a bug where `AllocDirStats` field was missing from Read Stats client API
```